runtime/src/bank/tests.rs

```diff
@@ -7724,7 +7728,16 @@ fn test_bpf_loader_upgradeable_deploy_with_max_len() {
         Vec::new(),
         Ok(()),
         solana_bpf_loader_program::process_instruction,
-        |_invoke_context| {},
+        |invoke_context| {
```

Comment:
This test need `programdata` account as well to load the program. The invoke context doesn't get the data account anymore, since we eagerly load the program in the bank. So we cannot use `solana_bpf_loader_program::test_utils::load_all_invoked_programs` here.